formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -382,8 +386,18 @@ type classAttributesOnKeys = {
   .
   [@bs.set] key1: string,
   /* The follow two are the same */
-  [@bs.get null] key2: [@onType2] Js.t(int),
-  [@bs.get null] key3: [@onType2] Js.t(int),
+  [@bs.get
+    {
```

Comment:
What's happening here then?

Reply:
The same thing. The attributes were specified with braces, and before this diff they were getting `refmt`ed away.

Reply:
Does that mean these will always get formatted like this now or something?

Reply:
If you remove the braces I guess they will get formatted how they were previously.

Reply:
Exactly. Previously, the braces were getting formatted away. Now, we preserve whether or not you included additional braces.